### ISSUE TITLE
swap Prev/Next widget & Help improve this page box

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-website#152]
So the Prev/Next widget is *above* the Help improve this page box

**Comment by yatil** [comment]
Hm. That would mean that screen reader and zoom users would leave to the next page before knowing that they can contribute. For me the help improve this page box is page-specific (so belongs to the content directly above it) while the prev/next is resource-specific (so belongs to all the pages in the prev/next flow). It feels unintuitive to me to first go to a resource specific element to go back to a page-specific element. 

That said, I don’t feel strongly.

**Comment by yatil** [comment]
Do in conjunction with #144.

**Comment by slhenry** [comment]
I agree with your points. I also think the additional points below related to the user experience are important.

We've had the Help improve this page box on all pages for a year. Very few people have used it (other than the Tutorials). I think that's just the nature of our material and readers. So it seems the usage data points to these not needing to have high billing. (except maybe in the tutorials)

In using the site recently, I totally missed the Prev/Next widget! Here's what happened -- that I think happens with others, too: Looking through several pages, my brain learned when I see the Help... box, it means: "Done with this page", and **brain doesn't look further**.

Later with my I-know-the-site-design hat on, I went looking at HowPWDsUseWeb to see how the Prev/Next widget was implemented in case it was easy to add it to the media resource I'm working on. I scrolled down... and didn't see it! Even though I knew it should be there. Because of the attention-grabbing weight of the teal buttons and the borders on both things and the alignment of the Previous and Next, it disappeared for me.
So I'm pretty sure some other people are going to miss it.

For some resources with the Previous/Next, we want to encourage readers to go to the next page -- or at least **know that it is available**. I think this is more important than the Help... box, especially given it's lack of use.

Also, most visual readers will still see the Help... box even if it's after Prev/Next because it stands out so much visually down there. And screen reader users who listen to the whole page or headers will also get it. It's an h2, so pretty prominent.

We could get input from EOWG and/or usability testing.
Or we could call it a minor issue that's not a priority for such input, and just do it.
